docs/guides/shared-cache.md

```diff
@@ -133,6 +133,21 @@ If a project has multiple CI pipelines or `earthly` invocations, it is recommend
 It is currently not possible to push both inline and implicit caches currently.
 {% endhint %}
 
+### Compatibility with major registry providers
+
+Not all registries support the needed manifest formats to allow the usage of `--remote-cache`. Here is a compatibility matrix for many popular registries:
```

Comment:
Oh - also I think both inline cache and explicit cache are affected. So the wording should be adjusted to make it clear that it applies to inline cache too.

Reply:
Looks like Inline and Remote are different enough to warrant a different column. Ill make that happen and then move it further up.